runzi/configuration/oq_opensha_nrml_convert.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pwd
+import itertools
+import stat
+import boto3
+from pathlib import PurePath
+
+import datetime as dt
+from dateutil.tz import tzutc
+
+
+from runzi.automation.scaling.toshi_api import SubtaskType
+from runzi.automation.scaling.python_task_factory import get_factory
+from runzi.util.aws import get_ecs_job_config
+
+import runzi.execute.openquake_hazard_task
+import runzi.execute.oq_opensha_convert_task
+
+from runzi.automation.scaling.local_config import (WORK_PATH, USE_API,
+    API_KEY, API_URL, CLUSTER_MODE, EnvMode )
+
+def build_hazard_tasks(general_task_id: str, subtask_type: SubtaskType, model_type: str, subtask_arguments):
+    task_count = 0
+    factory_class = get_factory(CLUSTER_MODE)
+
+    factory_task = runzi.execute.oq_opensha_convert_task
+    task_factory = factory_class(WORK_PATH, factory_task, task_config_path=WORK_PATH)
+
+    for source_gt_id in subtask_arguments['general_tasks']:
+
+        file_generator = get_output_file_ids(toshi_api, source_gt_id)
+        solutions = download_files(toshi_api, file_generator, str(WORK_PATH), overwrite=False,
+            skip_download=(CLUSTER_MODE == EnvMode['AWS']))
+
+        for (sid, solution_info) in solutions.items():
+
+            task_count +=1
+
+            # # The `tectonic_region_type` label must be consistent with what you use in the
+            # # logic tree for the ground-motion characterisation
+            # # Use "Subduction Interface" or "Active Shallow Crust"
+            # tectonic_region_type = "Subduction Interface"
+            if model_type == "CRUSTAL":
+                tectonic_region_type = "Active Shallow Crust"
+            elif model_type == "SUBDUCTION":
```

Comment:
@chrisbc do we want to split `SUBDUCTION` into Hikurangi and Puysegur?

Reply:
@chrisdicaprio converter needs to know the region_type only.